docs/release-notes/16-6-0/README.md

```diff
@@ -3,24 +3,103 @@ title: OpenProject 16.6.0
+
+**Scored lists** extend this concept by allowing administrators to assign numeric scores to list options, such as effort levels or risk ratings. These values can then be used as input for calculated results.
+
+**Calculated values** enable automatic computations based on formulas using numeric project attributes, including scores from Scored lists. The computed result is displayed directly on the project overview and in the project list. It automatically updates whenever one of its source attributes (e.g., *Benefit* or *Effort* in the example below) is changed. This allows teams to calculate project scores or priorities consistently across the portfolio.
+
+Here's an example of a calculated value called 'Overall score (calculated)' with the following formula: (Strategic impact * 0.6) + ( Benefit​* 0.3) - (Effort * 0.1)
+
+![Formula example for a project attribute of type 'Calculated Project Score': (Strategic impact * 0.6) + ( Benefit​* 0.3) - (Effort * 0.1)](openproject-16-6-project-attributes-calculated-value.png)
+
+Together, these additions give project portfolio managers a flexible foundation for data-driven project evaluation and more transparent portfolio decisions. 
+
+>[!NOTE]
+> Please note that these new project attributes are part of our [Enterprise add-ons in the Enterprise Professional plan](https://www.openproject.org/pricing/).
+
+![OpenProject 16.6: Project overview page showing three project attributes: Benefit = 10, Effort = 60, Overall Score (calculated) = automatically set to -1,2](openproject-16-6-project-attributes-overview-page.png)
+
+### Performance updates
+
+OpenProject 16.6 introduces several backend optimizations that significantly improve performance in large environments. [Database queries for collection endpoints in the API v3 have been optimized](https://community.openproject.org/wp/68457) to avoid unnecessary counting operations, and the [autocompleter for adding work package relations now requests only the data it actually needs](https://community.openproject.org/wp/68458).
+
+These improvements reduce query load and shorten response times, especially for installations with thousands of projects and work packages.
+
+### New index page for Documents module
+
+With OpenProject 16.6, a new index page provides a structured overview of all documents within a project. The list is sorted by last edited, showing the latest changes first, and includes columns for Name, Type, and Last edited. Users can search documents via a quick text filter or narrow results by document type using the filter menu on the left. 
+
+A new Create document button lets users quickly add new items, with Note set as the default type. Of course, the view automatically respects project permissions, ensuring that users only see documents they are allowed to access. On mobile, the list is optimized to show just the most relevant information.
```

Comment:
Probably a question of style but if the button is labeled 'Create document', I would put it in quotation marks. The button doesn't have the name but rather the label IMO.